contracts/base/LoanTermsConsensus.sol

```diff
@@ -22,11 +7,24 @@ import "./Consensus.sol";
 // Interfaces
 import "../interfaces/LoanTermsConsensusInterface.sol";
 
+/**
+    @notice This contract is used to process the loan requests through the Teller protocol
 
+    @author develop@teller.finance
+ */
 contract LoanTermsConsensus is Consensus, LoanTermsConsensusInterface {
+    /* Mappings */
     mapping(address => mapping(uint256 => ZeroCollateralCommon.AccruedLoanTerms)) public termSubmissions;
     mapping(address => mapping(uint256 => bool)) public requestNonceTaken;
 
+    /**
+        @notice Processes the loan request
+        @param ZeroCollateralCommon.LoanRequest request Struct of the protocol loan request
+        @param ZeroCollateralCommon.LoanResponses request List of structs of the protocol loan responses
+        @return uint256 Interest rate
```

Comment:
It should be @return interestRate Interest rate and the same for the other return params

Reply:
fixed